presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -181,7 +181,7 @@ queries.
        * Array checksum of the value set
 * **Row Columns**
     * Checksums row fields recursively according to the type of the fields.
-* For all other column types, generates a simple checksum using the :func:`checksum` function.
+* For all other column types, generates a simple checksum using the :func:`!checksum` function.
```

Comment:
what does adding the exclamation point do?

Reply:
My understanding is it's because we generate the links for `:func:` using some custom code, and so during initial rendering it throws a warning.  That is how I understood it from #23023